text/0107-construct-library-module-lifecycle.md

```diff
@@ -53,13 +60,14 @@ to add L2 resources in the future.
   - **Validate API reference** - Validate the module is listed in the AWS Construct Library API reference and is
     rendered correctly
 
-  - **Observe** - Watch KPIs to prioritize the construct library module's graduation:  
+### Ongoing activities
+  - **Observe** - Watch KPIs to prioritize the construct library module's graduation:
     - pull requests that add the first high-level constructs are the strongest signal to graduate a construct library
       module from stage 0 to stage 1 (experimental) and are prioritized first (this is a manual activity)
     - customer "+1s" on a construct library module's tracking issue exceeding 100 positive reactions, are prioritized
       next
     - construct library modules that accumulate 10+ feature requests, are prioritized next
-- **Exit Criteria** - AWS-CDK Construct Library modules move to stage 1:
+### Exit Criteria
   - when the first community PR for a module is merged, OR
   - when selected by the CDK team using the [prioritization framework](#Construct-prioritization-framework)
```

Comment:
Sorry this caught my eye. Can we just say, "when the first PR for this module that introduces constructs is merged"

Reply:
Agreed.